button-examples/Android/android-button-example.md

```diff
@@ -23,213 +21,387 @@ There are four types of buttons:
 3. [Contained button](#contained-button)
 4. [Toggle button](#toggle-button)
 
-![Example of the four button types](assets/button-types.svg)
+![Button types](assets/button-types.svg)
 
 ## Using buttons
 
-<!-- Update your IDE and your Material Components library to the latest version if you are using Material Components for the first time. -->
-
 Before you can use Material buttons, you need to add a dependency to the Material Components for Android library. For more information, go to the [Getting started](https://github.com/material-components/material-components-android/blob/master/docs/getting-started.md) page.
 
-### Making buttons accessible
-
-Android's button component APIs support labeling for accessibility. To use labels...
-
-For more guidance on writing labels, go to [our page on how to write a good accessibility label](https://material.io/design/usability/accessibility.html#writing).
-
-
 ## Text button
 
 [Text buttons](https://material.io/components/buttons/#text-button) are typically used for less-pronounced actions, including those located in dialogs and cards. In cards, text buttons help maintain an emphasis on card content.
 
-### Text button example
+### Text button examples
 
-Source code API:
+API and source code:
 
 * `MaterialButton`
     * [Class description](https://developer.android.com/reference/com/google/android/material/button/MaterialButton)
     * [GitHub source](https://github.com/material-components/material-components-android/blob/master/lib/java/com/google/android/material/button/MaterialButton.java)
 
 The following example shows a text button with a text label.
 
-!["Text button example for Android with purple text 'Text' over a white background."](assets/text-button.svg)
```

Comment:
The image descriptions are primarily for the vision impaired. We'll probably need to leave that in.

Reply:
Ah right, sorry, will add them back